src/solvers/dgsem_tree/dg_1d.jl

```diff
@@ -459,17 +461,19 @@ function calc_boundary_flux!(cache, t, boundary_conditions::NamedTuple,
 
   # Calc boundary fluxes in each direction
   calc_boundary_flux_by_direction!(surface_flux_values, t, boundary_conditions[1],
-                                   equations, surface_integral, dg, cache,
+                                   equations, have_nonconservative_terms(equations), surface_integral, dg, cache,
```

Comment:
Please use the order `have_nonconservative_terms(equations), equations`. That's what we do everywhere else. See https://trixi-framework.github.io/Trixi.jl/stable/styleguide/
(applies to several parts of the code)